b2/console_tool.py

```diff
@@ -736,7 +742,8 @@ def run(self, args):
                 content_type=args.contentType,
                 file_infos=file_infos,
                 sha1_sum=args.sha1,
-                progress_listener=progress_listener,
+                min_part_size=args.minPartSize,
+                progress_listener=progress_listener
```

Comment:
missing `,` at the end causes a 3-line diff instead of a 2-line diff (which is cleaner)